return-value of filecmp.dircmp.report_*

picibucor opened the pull request:
The functions: filecmp.dircmp.report() ,  filecmp.dircmp.report_partial_closure() , filecmp.dircmp.report_full_closure()
returning a dictionary with all the printed results

picibucor committed:
return-value of report_*

The functions: report() ,  report_partial_closure() , report_full_closure()
returning a dictionary with all the printed results

the-knights-who-say-ni commented:
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](http://bugs.python.org/) (b.p.o) to verify you have signed the CLA (this might be simply due to a missing "GitHub Name" entry in your b.p.o account settings). This is necessary for legal reasons before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


Thanks again to your contribution and we look forward to looking at it!

picibucor committed:
returning left- and right- paths separately

The filecmp.dircmp.report*  returns a dictionary 
left and right values returning in separate lists
